binnacle/android/build.gradle

```diff
@@ -1,11 +1,13 @@
 buildscript {
+    ext.kotlin_version = '1.3.21'
     repositories {
         google()
         jcenter()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.1'
+        classpath 'com.android.tools.build:gradle:3.3.2'
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
```

Comment:
Also, is this an environment variable?

Reply:
If it is, README needs to be updated with how to properly set up Kotlin stuff.

Reply:
I cloned locally, and I can't build the project because of Kotlin stuff. What's up?

Reply:
I removed the Kotlin stuff. Flutter analyze also runs cleanly now.